sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -15,6 +15,8 @@ dayjs.extend(dayOfYear);
 export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
 export const DATE_ONLY_FORMAT = "YYYY MMM DD";
 export const DATE_TIME_FORMAT = "YYYY-MM-DD HH:mm:ss";
+export const TIMESTAMP_CONTINUOUS_FORMAT = "YYYYMMDD-HHmmss";
+export const DATE_ONLY_DISPLAY_FORMAT = "MMM DD YYYY";
```

Comment:
Was the ISO format not enough to display the date properly during the CSV generation?